### ISSUE TITLE
[Settings]: 'Profile' text is not properly visible in High Contrast Desert Theme.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11257]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

Test Environment:
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Prerequisites:**
1. Open Settings app using 'Windows + i' Key.
2. Navigate to 'Accessibility' list item and activate it.
3. Then Navigate to 'Contrast themes' tab and activate it.
4. Apply Desert theme.

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Verify that 'Profile' text is properly visible in High Contrast Desert Theme or not.

**User Experience:**
User who user High contrast theme to use apps will be impacted here as they will not unable to see text if it is not visible properly.

**Guideline Reference:**
EN 301 549 V3.2.1 - 11.6.2.0 No disruption of accessibility features

**Attachments:**
[Profile text is not visible in High Contrast Desert Theme Which is visible in Normal theme and High Contrast Aquatic theme..zip](https://github.com/microsoft/terminal/files/7183582/Profile.text.is.not.visible.in.High.Contrast.Desert.Theme.Which.is.visible.in.Notmal.theme.and.High.Contrast.Aquatic.theme.zip)

### Expected Behavior

Profile text should visible properly in High Contrast Desert Theme Which is visible in Normal theme and High Contrast Aquatic theme..

### Actual Behavior

Profile text is not visible in High Contrast Desert Theme Which is visible in Normal theme and High Contrast Aquatic theme..

**Comment by zadjii-msft** [comment]
![Profile text is not visible in High Contrast Desert Theme Which is visible in Notmal theme and High Contrast Aquatic theme](https://user-images.githubusercontent.com/18356694/147975937-763bdb9e-5886-4a12-9cc1-52e3d404afa4.png)

This sure looks like something that should be fixed upstream in WinUI - we're not providing any custom styling for the NavigationView for Desert Theme, which would suggest that any apps that are using the NavigationView in WinUI would similarly face this issue. @v-rpundir can we move this to an upstream issue on https://github.com/microsoft/microsoft-ui-xaml?

**Comment by v-rpundir** [comment]
@zadjii-msft Yup you can move this issue to https://github.com/microsoft/microsoft-ui-xaml

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
